extensions/amp-carousel/0.1/test/test-slides.js

```diff
@@ -382,8 +382,8 @@ describe('Slides functional', () => {
         pos: 0.6,
         min: 0,
         max: 0,
-        prevTr: prevTr,
-        nextTr: nextTr,
+        prevTr,
```

Comment:
honestly i don't have any strong opinion on this. since we can't enforce it right now anyways i do think we should just leave it as is